doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrators
+======================
+
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands, along with an analysis and
+visualization workflow connected to the simulation. In such cases, SmartSim
+allows you to augment the number of clustered orchestrators or provision multiple
+co-located orchestrators, ensuring that the workload can scale to meet its resource
+requirements. Below is a general example demonstrating the process of launching
+multiple databases with unique identifiers and distributing data between them.
+
+.. note::
+
+  To run the following example, configure your environment by following
```

Comment:
I don't think you need this note. Also, users might either be using a site install or need to go through the "Basic Installation" so this note is somewhat misleading.